### PR TITLE
Update wording + add `,` as key for major mode

### PR DESCRIPTION
**Pull request by stevenguh**
This is mostly for https://github.com/VSpaceCode/VSpaceCode/commit/a0060b5b8a104de994735515d74f9f2b3d18b9a0 and not being released yet.

### CODE REVIEW

File: docs/major-mode.md [hunk 1]
```diff
@@ -3,19 +3,15 @@ id: major-mode
 title: Major Mode
 ---
 
-In the Visual Studio Code status bar, on the right, you can find the currently active language mode:
+Major mode offers dynamic menu and functions based on the activate buffer. You can access key bindings specific to the current major mode by pressing `<spc> m`. In addition, `,` is bound to major mode menu to save one keystroke.
```
Comment (marcoieni) [inline docs/major-mode.md]:
```suggestion
Major mode offers dynamic menu and functions based on the active buffer. You can access key bindings specific to the current major mode by pressing `<spc> m` or `,` if you want to save one keystroke.
```

File: docs/major-mode.md [hunk 2]
```diff
@@ -3,19 +3,15 @@ id: major-mode
...
-![major mode position](/img/docs/major_mode.png)
-
-In order to follow emacs terminology, we call it major mode.
-
-You can access key bindings specific to the current major mode by pressing `<spc m>`.
 For example, you can press `<spc> m x b` to make the text bold in a markdown buffer.
 
-The vast majority of major mode specific key bindings will require the installation of additional extensions
-in order to work properly.
+To find the activate language mode of your current buffer, you can look the bottom right on the status bar.
```
Comment (marcoieni) [inline docs/major-mode.md]:
```suggestion
To find the active language mode of your current buffer, you can look the bottom right of the status bar.
```

File: docs/major-mode.md [hunk 3]
```diff
@@ -3,19 +3,15 @@ id: major-mode
...
+
+![major mode position](/img/docs/major_mode.png)
 
-The table below lists all the extensions required by each language in the major mode.
+The vast majority of major mode specific key bindings will require the installation of additional extensions in order to work properly. The section below lists all the extensions required by each language in the major mode.
```
Comment (marcoieni) [inline docs/major-mode.md]:
```suggestion
The vast majority of major mode specific key bindings will require the installation of additional extensions in order to work properly. The section below lists all the extensions required by each major mode.
```

**Comment by stevenguh** [comment]
btw, the suggestion feature is kind of cool :)

**Comment by marcoieni** [comment]
Yes :)
Anyway, this looks good to me. We can merge this when the actual feature is shipped.

**Comment by stevenguh** [comment]
Per https://github.com/VSpaceCode/vspacecode.github.io/issues/8#issuecomment-705720440. Trying out the develop branch approach :) Let's see if this works well.

**Pull request merged**